src/pages/Main/Main.scss

```
@@ -0,0 +1,104 @@
+@import url('https://fonts.googleapis.com/css2?family=Noto+Sans+KR&display=swap');
+* {
+  font-family: 'Noto Sans KR', sans-serif;
+  font-size: 18px;
+}
+.main {
+  display: flex;
+  flex-direction: column;
+}
+.promotion {
+  background-color: rgb(244, 244, 244);
+  .promotionPhrase {
+    margin: 40px;
+    font-size: 40px;
+    font-weight: bold;
+  }
+  .promotionBtn {
+    margin-left: 40px;
+    margin-bottom: 40px;
+    width: 200px;
+    height: 50px;
+    border: 0;
+    border-radius: 5px;
+    background-color: #201c1d;
+    color: white;
+    font-size: 18px;
+  }
+}
+.delivery {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  .deliveryLayout {
+    display: flex;
+    flex-direction: row;
```

`flex-direction: row`는 flex의 디푤드값이여서 굳이 명시하지 않아도 됩니다.
여기서,
```suggestion
    display: flex;
    align-items: center;
    justify-content: center;
```
주면 될 것 같네요.